### PR TITLE
Add a link to the Guideline for translating JASP in the help file

### PR DESCRIPTION
**Pull request by boutinb**
Fixes https://github.com/jasp-stats/jasp-test-release/issues/1537

### CODE REVIEW

**Comment by amirmasoudabdol** [comment]
@boutinb, we can maybe add the same message and link under the Choose Language dropdown in the Preferences. This might be helpful bringing new translators to us. It may also save us from some translation-related issues on GitHub when people find something is missing. 

Something like this mockup:

Screenshot 2021-09-15 at 13 47 53

**Comment by boutinb** [comment]
Yes I was thinking of that also, but that would mean that this string has to be also translated in all languages we support now (Dutch, German, Portuguese, Spanish, Galician, Chinese, Japanese) for 0.15... I can ask them to have a last look again this week, but then we should not add strings anymore.

**Comment by amirmasoudabdol** [comment]
That's true! I guess, I just adjust the UI, and we leave it until we have the translation for it.

**Comment by JoKeyser** [comment]
My comment about the link location applies here too, please see https://github.com/jasp-stats/jasp-desktop/pull/4626#issuecomment-920089610

File: Resources/Help/preferences/prefsui.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ Here you can specify if you would like to have a "light" or a "dark" theme on th
 ### Preferred Language
 Select the language in which you would like to use JASP.
 
+If you would like to improve one translation, or to add a new language, please check this document: https://github.com/jasp-stats/jasp-desktop/blob/development/Docs/development/jasp-guideline-translators.md
```
Comment (TimKDJ) [inline Resources/Help/preferences/prefsui.md]: I think the text used in https://github.com/jasp-stats/jasp-desktop/pull/4625#issuecomment-919948474 works a bit better. Also good to use the jasp-stats.org url here: https://jasp-stats.org/translation-guideline

**Review by TimKDJ — CHANGES_REQUESTED** [review]

File: Resources/Help/preferences/prefsui.md (hunk 1 above)
Comment (AlexanderLyNL) [reply Resources/Help/preferences/prefsui.md]: I tend to copy paste this md into https://jasp-stats.org/translation-guidelines

**Comment by boutinb** [comment]
I have made the changes. If @AlexanderLyNL agree, we can merge it

**Review by AlexanderLyNL — APPROVED** [review]
Seems good to me. I didn't follow the whole thread, but hopefully TIm is okay with this as well now.

**Pull request merged**